### PR TITLE
Parallel tests should require the 3rd edition

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: R/parallel-config.R [hunk 1]
```diff
@@ -14,5 +14,18 @@ find_parallel <- function(path, load_package = "source", package = NULL) {
...
+  par <- identical(
+    toupper(desc$get_field("Config/testthat/parallel", default = "FALSE")),
+    "TRUE"
+  )
+
+  if (par) {
+    ed <- as.integer(desc$get_field("Config/testthat/edition", default = 2L))
+    if (ed < 3) {
+      inform("Running tests in parallel requires the 3rd edition")
+      want_parallel <- FALSE
```
Comment (hadley) [inline R/parallel-config.R:26]: Should be `return(FALSE)` ?

Comment (gaborcsardi) [reply R/parallel-config.R:26]: oh, sorry, it should be 
```
par <- FALSE
```

good catch.

Comment (gaborcsardi) [reply R/parallel-config.R:26]: should this be a warning, btw?

Comment (gaborcsardi) [reply R/parallel-config.R]: Anyway, should be good now.

Comment (hadley) [reply R/parallel-config.R]: I don't feel too strongly about it either way

Comment (gaborcsardi) [reply R/parallel-config.R]: Me neither, so let's leave it as is. :)

**Pull request merged**